roles/kubernetes-apps/ansible/templates/netchecker-agent-ds.yml.j2

```diff
@@ -40,6 +40,12 @@ spec:
             requests:
               cpu: {{ netchecker_agent_cpu_requests }}
               memory: {{ netchecker_agent_memory_requests }}
+          securityContext:
+            runAsUser: {{ netchecker_agent_user | default('0') }}
+            runAsGroup: {{ netchecker_agent_group | default('0') }}
+{% if rbac_enabled %}
```

Comment:
create the service account and roles even if the rbac is not enabled. it will just be ignored.

Reply:
Right I saw the PR but prefered to wait until it is merged. I'm on it